vaadin-icons-flow-parent/vaadin-icons-flow-integration-tests/src/main/java/com/vaadin/flow/component/icon/demo/IconView.java

```diff
@@ -15,27 +15,27 @@
  */
 package com.vaadin.flow.component.icon.demo;
 
+import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
 import com.vaadin.flow.component.html.Label;
 import com.vaadin.flow.component.icon.Icon;
 import com.vaadin.flow.component.icon.IronIcon;
 import com.vaadin.flow.component.icon.VaadinIcon;
 import com.vaadin.flow.component.orderedlayout.FlexComponent.Alignment;
 import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
-import com.vaadin.flow.demo.DemoView;
 import com.vaadin.flow.router.Route;
 
 /**
  * View for {@link Icon} demo.
  *
  * @author Vaadin Ltd
  */
-@Route("vaadin-icons")
-public class IconView extends DemoView {
+@Route("vaadin-icons/icon-view")
+public class IconView extends Div {
 
-    @Override
-    public void initView() {
+    public IconView() {
```

Comment:
This test view includes a `new IronIcon("lumo", "clock")`, which is not rendered correctly (see `createBasicIconsView()` below).
1. Is the use of an `IronIcon` here intentional?
2. Why is it not rendered? Is the Lumo icon set loaded?

Reply:
Seems that the IronIcon is there intentionally as it's tested in the ITs. The import of lumo-icons was missing. Added it in https://github.com/vaadin/flow-components/pull/1785/commits/0b3e0be2d7dcb4b000df73281ff4724f7755a49b

Reply:
Ah, just noticed that the lumo-icons now also include vaadin-icon -based icons. Updated the test once again https://github.com/vaadin/flow-components/pull/1785/commits/6822d9dee80a58df7d993cf9c4385db0b27ad9cb